### ISSUE TITLE
[Fabric 1.20.5] Smithing Table Crash

### ISSUE THREAD

**Issue by Basdower** [Craftjakob/Enderite-CJ-Edition#9]
It´s a really good mod! However I started a world on 1.20.5 and got to the point of upgrading to enderite and when I put the three items on the smithing table the game crashes under the error "The game crashed whilst container click Error: java.lang.UnsupportedOperationException". I tried checking if it was incompatible with some other mod but even being the only mod the crash still happens.

Edit: It also seems I can't combine enderite armor with the elytra

**Comment by Craftjakob** [comment]
The reason was, that the ItemEnchantments returns an Unmodifiable Map, that I want to modify.
But now I have fixed that.
![image](https://github.com/Craftjakob/Enderite-CJ-Edition/assets/107345776/c8e600e7-b333-4594-aa25-55fa73738278)

Yes, the Enderite Elytra is not Craftable.
And I have also now fixed it.

The Problem is, that Minecraft has changed a lot in their code.

In the next update, everything is fixed.
